03-specifying-sample-designs.Rmd

```diff
@@ -62,7 +64,7 @@ des_srs1 <- dat %>%
  as_survey_design(fpc = fpcvar)
 ```
 
-where `dat` is a tibble or data.frame with the survey data, and `fpcvar` is a variable on the tibble indicating the sampling frame's size. If the frame is very large, sometimes the frame size is not provided. In that case, the fpc is not needed, and specify the design as:
+where `dat` is a tibble or data.frame with the survey data, and `fpcvar` is a variable on the tibble indicating the sampling frame's size. If the frame is very large, sometimes the frame size is not provided. In that case, the FPC is not needed, and specify the design as:
```

Comment:
> If the frame is very large

There is a "cutpoint" for what is considered large?  If so, we should consider adding that here.